org.eclipse.xtext.ui.tests/src-gen/org/eclipse/xtext/ui/tests/editor/contentassist/services/Bug287941TestLanguageGrammarAccess.java

```diff
@@ -411,19 +412,19 @@
 		private final RuleCall cEntriesAndWhereEntryParserRuleCall_1_1_1_0 = (RuleCall)cEntriesAssignment_1_1_1.eContents().get(0);
 		
 		//WhereEntry:
-		//	AndWhereEntry ({OrWhereEntry.entries+=current} ("or" entries+=AndWhereEntry)+)?;
+		//	AndWhereEntry ({OrWhereEntry.entries+= current} ("or" entries+=AndWhereEntry)+)?;
```

Comment:
Still an unexpected space after the assignment op @ArneDeutsch

Reply:
Created a unit test, but there it works as expected ...

Reply:
when i format

``` 
AndWhereEntry returns WhereEntry:
	ConcreteWhereEntry ({AndWhereEntry.entries += current}
	("and" entries+=ConcreteWhereEntry)+)?;
``` 

spaces dont get removed

Reply:
Will be fixed with next commit. Only formatted "=" but missed "+=".